__tests__/__utils__/services/frontend.ts

```diff
@@ -82,6 +82,8 @@ export function defaultFrontendServer(): RestResolver {
       content = 'Edit Page'
     } else if (req.url.pathname === '/en/event/history/user/1/arekkas') {
       content = 'arekkas'
+    } else if (req.url.pathname === '/en/content-only/1555') {
+      content = 'Article (no header)'
```

Comment:
Is there a way to determine that it is a content-only response of the frontend?

Reply:
If frontend returns with content when queried for `/content-only/` we can be pretty sure it's content only.
Otherwise: Check if `#__next > header` returns something.